src/App.js

```diff
@@ -48,21 +62,61 @@ const App = () => {
     // 3. Go across each diagonal to see if 
     //    all three squares have the same value.
 
+    // check vertically
+    for (let i = 0; i < 3; i++) {
+      if (squares[0][i].value === squares[1][i].value &&
+        squares[1][i].value === squares[2][i].value &&
+        squares[0][i].value !== '') {
+          return squares[0][i].value
+        }
+    }
+    // check horizontally
+    for (let i = 0; i < 3; i++) {
+      if (squares[i][0].value === squares[i][1].value &&
+        squares[i][1].value === squares[i][2].value &&
+        squares[i][0].value !== '') {
+          return squares[i][0].value
+        }
+    }
+    // check diagonally
+    if (squares[0][0].value === squares[1][1].value &&
+      squares[1][1].value === squares[2][2].value &&
+      squares[0][0].value !== '') {
+        return squares[0][0].value
+    }
+    if (squares[0][2].value === squares[1][1].value &&
+      squares[1][1].value === squares[2][0].value &&
+      squares[0][2].value !== '') {
+        return squares[0][2].value
+      }
+      return null;
+  }
+
+  const printWinner = () => {
+    if (winner == null) {
```

Comment:
Because of the explicit `null` check, the first move each round doesn't display the player's turn info. Either change the value used for winner at the start of a game to `null`, or change this check to winner being falsy 

```js
if (! winner) {
```

since both `null` and empty string are falsy.